web_monitoring/internetarchive.py

```diff
@@ -34,6 +34,24 @@ class UnexpectedResponseFormat(WebMonitoringException):
 URL_CHUNK_PATTERN = re.compile('\<(.*)\>')
 DATETIME_CHUNK_PATTERN = re.compile(' datetime="(.*)",')
 
+def check_exists(lines):    
+    """
+    Check if Internet Archive has archived versions of a url.
+    """
+
+
+    try:
+        # The first three lines contain no information we need.
+        for _ in range(3):
+            next(lines)
+            
+
+    except StopIteration:
+        print("Internet archive does not have archived versions of this url.")
```

Comment:
Alright, I'll change it to a comment. I guess I'll learn more about good practices for library code with experience.